src/StrongNameSign.targets

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<ItemGroup>
+		<StrongNameSignPackage Include="Markdig"/>
+		<StrongNameSignPackage Include="Markdig.Wpf"/>
+	</ItemGroup>
+
+	<Target Name="BeforeBuild">
+		<Message Text="Running StrongNameSigner on packages: @(StrongNameSignPackage)..." Importance="high"/>
+		<Exec ContinueOnError="false"
+			  Command="&quot;$(MSBuildThisFileDirectory)..\packages\Brutal.Dev.StrongNameSigner.2.1.3\build\StrongNameSigner.Console.exe&quot; -in &quot;@(StrongNameSignPackage->'$(MSBuildThisFileDirectory)..\packages\%(Identity).*', '|')&quot;"/>
+	</Target>
```

Comment:
Maybe put this next to the other targets files in the src/common folder, so they're all in one place?